db/pre-re/pet_db.conf

```diff
@@ -0,0 +1,1544 @@
+//=      | | | |                   | |
+//=      | |_| | ___ _ __ ___ _   _| | ___  ___
+//=      |  _  |/ _ \ '__/ __| | | | |/ _ \/ __|
+//=      | | | |  __/ | | (__| |_| | |  __/\__ \
+//=      \_| |_/\___|_|  \___|\__,_|_|\___||___/
+//================= License ===============================================
+//= This file is part of Hercules.
+//= http://herc.ws - http://github.com/HerculesWS/Hercules
+//=
+//= Copyright (C) 2018  Hercules Dev Team
+//=
+//= Hercules is free software: you can redistribute it and/or modify
+//= it under the terms of the GNU General Public License as published by
+//= the Free Software Foundation, either version 3 of the License, or
+//= (at your option) any later version.
+//=
+//= This program is distributed in the hope that it will be useful,
+//= but WITHOUT ANY WARRANTY; without even the implied warranty of
+//= MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//= GNU General Public License for more details.
+//=
+//= You should have received a copy of the GNU General Public License
+//= along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//=========================================================================
+//= Pets Database
+//=========================================================================
+
+pet_db:(
+/**************************************************************************
+ ************* Entry structure ********************************************
+ **************************************************************************
+{
+	// ================ Mandatory fields ==============================
+	Id: ID                               (int)
+	SpriteName: "Sprite_Name"            (string)
+	Name: "Pet Name"                     (string)
+	// ================ Optional fields ===============================
+	TamingItemID: Taming Item            (string, defaults to 0)
+	EggItemID: Egg Id                    (string, defaults to 0)
+	AccessoryItemID: Equipment Id        (string, defaults to 0)
+	FoodItemID: Food Id                  (string, defaults to 0)
+	Fullness: hunger points              (int, defaults to 0)
+	HungryDelay: hunger time             (int, defaults to 0)
+	Intimacy: {
+		Start: start intimacy            (int, defaults to 0)
+		Feed: feeding intimacy           (int, defaults to 0)
+		OverFeed: overfeeding intimacy   (int, defaults to 0)
+		OwnerDie: owner die intimacy     (int, defaults to 0)
+	}
+	CaptureRate: capture rate            (int, defaults to 0)
+	Speed: speed                         (int, defaults to 0)
+	SpecialPerformance: true/false       (boolean, defaults to false)
+	TalkConvertClass: convert talk       (int, defaults to 0)
+	AttackRate: attack rate              (int, defaults to 0)
+	DefenceAttackRate: Defence attack    (int, defaults to 0)
+	ChangeTargetRate: change target      (int, defaults to 0)
+	PetScript: <" Pet Script (can also be multi-line) ">
+	EquipScript: <" Equip Script (can also be multi-line) ">
+},
```

Comment:
I'd like to use this chance to change the spelling of some of these keys (because it gets more annoying to do this later, once people have converted their databases). My suggested keys are:

- `Id`: Ok.
- `SpriteName`: Ok
- `Name`: Ok
- `TamingItemID`: I'd shorten this to `TamingItem` to hint that we don't ask for a number
- `EggItemID`: As above, `EggItem`
- `AccessoryItemID`: As above, `AccessoryItem`
- `FoodItemID`: As above, `FoodItem`
- `Fullness`: I'd change this to the more meaningful `FoodEffectiveness`
- `HungryDelay`: I'd correct this to `HungerDelay`
- `Intimacy/Start`: I'd perhaps change this to `Intimacy/Initial`, or `Intimacy/Starting` if you prefer
- `Intimacy/Feed`: I'd clarify this as `Intimacy/FeedIncrement`
- `Intimacy/OverFeed`: Same as above, `Intimacy/OverFeedDecrement`
- `Intimacy/OwnerDie`: Same as above, `Intimacy/OwnerDeathDecrement`
- `CaptureRate`: Ok
- `Speed`: Ok
- `SpecialPerformance`: Ok
- `TalkConvertClass`: This should probably be a boolean, named `TalkWithEmotes` or `UseEmotes`
- `AttackRate`: Ok
- `DefenceAttackRate`: Should be corrected to `DefendRate` (considering "attack" above as a verb)
- `ChangeTargetRate`: Ok
- `PetScript`: Ok
- `EquipScript`: Ok